packages/sdk/src/types/Rewards.ts

```diff
@@ -0,0 +1,92 @@
+export interface RewardsListParams {
+	page?: number
+	limit?: number
+}
+
+export interface RewardsListResponse {
+	object: 'list'
+	total: number
+	data_ref?: string
+	data: {
+		id?: string
+		name?: string
+		parameters?: {
+			campaign?: {
+				id?: string
+			}
+		}
+		created_at?: string
+		updated_at?: string
+		object?: string
+	}[]
+}
+
+export interface RewardsCreate {
+	name?: string
+	parameters?: {
+		campaign?: {
+			id?: string
+		}
+	}
+}
+
+export interface RewardsCreateResponse {
+	id?: string
+	name?: string
+	created_at?: string
+	parameters?: {
+		automation_id?: string
+		campaign?: {
+			id?: string
+			type?: string
+		}
+	}
+	object?: 'reward'
+}
+
+export type RewardsGetResponse = RewardsCreateResponse
+
+export type RewardsUpdate = RewardsCreate & { id?: string }
```

Comment:
It's like Create but you can't overwrite `type` property so please fix as according to comments above and just make sure that "type" can not be passed. Also why `id` has `?` ? For me, it seems like required property in order to update.